stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamColors.kt

```diff
@@ -47,6 +47,10 @@ import io.getstream.chat.android.compose.R
  * @param threadSeparatorGradientEnd Used as an end color for vertical gradient background in a thread separator.
  * @param ownMessageText Used for message text color for the current user. [textHighEmphasis] by default.
  * @param otherMessageText Used for message text color for other users. [textHighEmphasis] by default.
+ * @param ownMessageQuotedBackground Used as a quoted section background color for the messages sent by the current user.
```

Comment:
What do you think about something like this?
It's just a suggestion, feel free to dismiss it.

```suggestion
 * @param ownMessageQuotedBackground Changes the background color of the quoted message contained in a reply sent by the current user.
```